src/tests/multihost/alltests/test_sss_cache.py

```diff
@@ -0,0 +1,60 @@
+""" Automation of sss_cache tests
+
```

Comment:
Add the requirement and casecomponent here
eg. https://github.com/SSSD/sssd/blob/master/src/tests/multihost/alltests/test_kcm.py#L3

Reply:
Done